openshift/README.md

```diff
@@ -30,3 +42,10 @@ CentosCI will build your image and comment on the PR with an image name and tag.
 Your image is available in the registry: docker pull registry.devshift.net/fabric8-analytics/worker-scaler:SNAPSHOT-PR-25
 
 To update your dev deployment use [oc edit](https://docs.openshift.com/container-platform/3.4/cli_reference/basic_cli_operations.html#edit) from the command line or editor in web interface.
+
+## Run E2E test agaist your deployment
+
+Environment variables are set by running deploy.sh.
```

Comment:
Wouldn't be more fool-proof approach here to directly source `env.sh` inside `run-2e2-test.sh`?

i.e.:
```
. env.sh
```